src/Core/Enums/ImageFit.cs

```diff
@@ -0,0 +1,31 @@
+﻿// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// Determines how the image will be scaled and positioned within its parent container.
+/// </summary>
+public enum ImageFit
+{
+    /// <summary>
+    /// The image will not be resized or repositioned.
+    /// </summary>
+    None,
```

Comment:
You need to add an attribute `Description` for ALL enum values. And to use `ToAttributeValue` to get this value in your razor file.